services/brig/src/Brig/API/User.hs

```diff
@@ -207,9 +207,16 @@ createUser new = do
             Nothing
           )
         Just existingAccount ->
-          let ident = newIdentity email phone (newUserSSOId new)
-              existingUser = accountUser existingAccount
-           in ( new {newUserManagedBy = Just (userManagedBy existingUser), newUserIdentity = ident},
+          let existingUser = accountUser existingAccount
+              mbSSOid =
+                case (teamInvitation, email, userManagedBy existingUser) of
+                  (Just _, Just em, ManagedByScim) ->
```

Comment:
leave a comment that `isJust teaminvitation` implies that there is no idp?

Reply:
Done